doc/modules/calibration.rst

```diff
@@ -111,30 +123,68 @@ predicted probabilities of the `k` estimators in the
 `calibrated_classifiers_` list. The output of :term:`predict` is the class
 that has the highest probability.
 
-The regressor that is used for calibration depends on the `method`
-parameter. `'sigmoid'` corresponds to a parametric approach based on Platt's
-logistic model [3]_, i.e. :math:`p(y_i = 1 | f_i)` is modeled as
-:math:`\sigma(A f_i + B)` where :math:`\sigma` is the logistic function, and
-:math:`A` and :math:`B` are real numbers to be determined when fitting the
-regressor via maximum likelihood. `'isotonic'` will instead fit a
-non-parametric isotonic regressor, which outputs a step-wise non-decreasing
-function (see :mod:`sklearn.isotonic`).
-
-An already fitted classifier can be calibrated by setting `cv="prefit"`. In
-this case, the data is only used to fit the regressor. It is up to the user
+Alternatively an already fitted classifier can be calibrated by setting
+`cv="prefit"`. In this case, the data is not split and all of it is used to
+fit the regressor. It is up to the user
 make sure that the data used for fitting the classifier is disjoint from the
 data used for fitting the regressor.
 
-:class:`CalibratedClassifierCV` can calibrate probabilities in a multiclass
-setting if the base estimator supports multiclass predictions. The classifier
-is calibrated first for each class separately in a one-vs-rest fashion [4]_.
-When predicting probabilities, the calibrated probabilities for each class
+:class:`CalibratedClassifierCV` supports the use of two 'calibration'
+regressors: 'sigmoid' and 'isotonic'. Both these regressors only
+support 1-dimensional data (e.g., binary classification output) but are
+extended for multiclass classification if the `base_estimator` supports
+multiclass predictions. For multiclass predictions,
+:class:`CalibratedClassifierCV` calibrates for
+each class separately in a :ref:`ovr_classification` fashion [4]_. When
+predicting
+probabilities, the calibrated probabilities for each class
 are predicted separately. As those probabilities do not necessarily sum to
 one, a postprocessing is performed to normalize them.
 
 The :func:`sklearn.metrics.brier_score_loss` may be used to evaluate how
 well a classifier is calibrated.
 
+Sigmoid
+^^^^^^^
+
+The sigmoid regressor is based on Platt's logistic model [3]_:
+
+.. math::
+       p(y_i = 1 | f_i) = \frac{1}{1 + \exp(A f_i + B)}
+
+where :math:`y_i` is the true label of sample :math:`i` and :math:`f_i`
+is the output of the classifier for sample :math:`i`. :math:`A` and :math:`B`
```

Comment:
just to avoid any potential ambiguity

```suggestion
is the output of the un-calibrated classifier for sample :math:`i`. :math:`A` and :math:`B`
```